app/views/search/search.html.erb

```diff
@@ -1,3 +1,6 @@
+<% content_for :switcher do %>
```

Comment:
I wonder if 
```
<% content_for :switcher do %>
  <%= render '/layouts/prison_switcher' %>
<% end %>
```

can be moved into layouts/application.html.erb  - then it's only defined in one place